.env.example

```diff
@@ -1,5 +1,9 @@
 PRIVATE_KEY=<your private key>
 
+# All network RPCs can be populated with the newest and most reliable RPCs fetched from MongoDB
+# These values are stored to share the most up-to-date RPCs that work, ensuring consistency across all team members and github actions
+# To populate all RPCs, please run "bun fetch-rpcs"
+# To add a new RPC for a specific network, run: "bun add-network-rpc --network {networkName} --rpcUrl {rpcUrl}"
 # Mainnet RPCs (the 'pre-commit-checker' comment after the URL is only for our internal tool and not required in your .env file)
 ETH_NODE_URI_MAINNET=https://1rpc.io/eth #[pre-commit-checker: not a secret]
```

Comment:
Should we remove these from the example `.env` as well?

Reply:
I was thinking the same, on one hand, it’s helpful to show that these will be included in `.env`, but if they’re generated automatically, you’re right that it doesn’t really make sense to include them in .env.example. Removing them also means one less thing to maintain when adding support for a new chain. I’ve gone ahead and removed them